synapse/rest/client/v1/register.py

```diff
@@ -128,12 +128,10 @@ def on_POST(self, request):
             login_type = register_json["type"]
 
             is_application_server = login_type == LoginType.APPLICATION_SERVICE
-            is_using_shared_secret = login_type == LoginType.SHARED_SECRET
```

Comment:
can we deprecate this for now, rather than remove it, please?

Reply:
done